charts/ehrenamt-justiz/templates/imagestream.yml

```
@@ -0,0 +1,21 @@
+{{- if $.Values.imagestream }}
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: {{ include "getFullnameImageStream" (list $.Release.Name $.Values.imagestream) }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)